### PR TITLE
Build lifecycle image for Windows and create manifest list

### PR DESCRIPTION
**Pull request by natalieparellano**
Tested here: https://github.com/natalieparellano/lifecycle/actions/runs/196468263

It created the images here: https://hub.docker.com/repository/docker/natalieparellano/lifecycle/tags?page=1

Tested them out on both platforms, they work :)

### CODE REVIEW

File: .github/workflows/lifecycle-image.yml [hunk 1]
```diff
@@ -17,6 +17,33 @@ jobs:
...
+          $LIFECYCLE_VERSION=$(echo ${{ github.event.release.tag_name }} | cut -d "v" -f2)
+          curl -o lifecycle.tgz -L https://github.com/buildpacks/lifecycle/releases/download/v${LIFECYCLE_VERSION}/lifecycle-v${LIFECYCLE_VERSION}+windows.x86-64.tgz
+          echo ${{ secrets.DOCKER_PASSWORD }} | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
+          go run ./tools/image/main.go -lifecyclePath ./lifecycle.tgz -tag buildpacksio/lifecycle:${LIFECYCLE_VERSION}-windows -tag buildpacksio/lifecycle:latest-windows
+  create-lifecycle-image-manifest-list:
+    needs: [build-lifecycle-image-linux, build-lifecycle-image-windows]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Create Manifest Life
```
Comment (ekcasey) [inline .github/workflows/lifecycle-image.yml:42]:
```suggestion
      - name: Create Manifest List
```

File: .github/workflows/lifecycle-image.yml [hunk 2]
```diff
@@ -17,6 +17,33 @@ jobs:
...
+        run: |
+          LIFECYCLE_VERSION=`echo ${{ github.event.release.tag_name }} | cut -d "v" -f2`
+          echo ${{ secrets.DOCKER_PASSWORD }} | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
+          DOCKER_CLI_EXPERIMENTAL=enabled docker manifest create buildpacksio/lifecycle:0.99.20 buildpacksio/lifecycle:0.99.20-linux buildpacksio/lifecycle:0.99.20-windows
+          DOCKER_CLI_EXPERIMENTAL=enabled docker manifest push buildpacksio/lifecycle:0.99.20
```
Comment (ekcasey) [inline .github/workflows/lifecycle-image.yml:47]:
```suggestion
          DOCKER_CLI_EXPERIMENTAL=enabled docker manifest create buildpacksio/lifecycle:${LIFECYCLE_VERSION} buildpacksio/lifecycle:${LIFECYCLE_VERSION}-linux buildpacksio/lifecycle:${LIFECYCLE_VERSION}-windows
          DOCKER_CLI_EXPERIMENTAL=enabled docker manifest push buildpacksio/lifecycle:${LIFECYCLE_VERSION}
```

Comment (natalieparellano) [reply .github/workflows/lifecycle-image.yml]: 🤦  good catch...

Pulling in the commit suggestion seems to have broken DCO... do you know how to fix?

Comment (ekcasey) [reply .github/workflows/lifecycle-image.yml]: You need to pull down the change `git rebase` with `--signoff` and force push it. I have definitely done this before, you can add suggestions w/o breaking DCO by copying you signoff into the commit in the UI.

**Pull request merged**